### PR TITLE
Refactor inventory system

### PR DESCRIPTION
**Pull request by 102113528**
This PR implements the inventory system via the following commits:

- Implement the base inventory system with input callbacks, select, pick up, use, and drop interactions (a44eb8ace2791d02ff5f813e4d09e0c26be4ba19).
- Document important aspects of the inventory system (fc780afe7e991d85d6d303eecd737eb97695cd32).
- Prevent items from being dropped when they're active (0d8794d4c6588ca00d43675bef619f4ea00cef35).
- Implement an item picker indicator for the currently selected item in the floor (7ab32de4e33aaf823a1b6f8fd0a9690332abda18).
- Implement basic animations for pick up, use, and dropping of items (c398d5f685ad103d9c70055f8f09eb73c8946ac2).
- Don't attempt to pick up nearby items if there aren't any (c2809a5285b41bd10a631bbf17e5130baaeb5e3d).
- Implement speed increase item as an initial test item (e57b00237b39c193f764e7280a2e877eaa6bc145).

To test, drag the player, item speed increase, and level prefabs into the root scene, and drag the inventory prefab onto the main camera. When running around, the item picker should show above nearby items, and pressing SHIFT + 1/2/3/4 should equip the item into the respective slot. Pressing 1/2/3/4 should activate the item in that slot. Once the item has finished being used, it should disappear from the slot.

### CODE REVIEW

File: MicrowaveGame/ProjectSettings/ProjectSettings.asset [hunk 1]
```diff
@@ -630,7 +630,7 @@ PlayerSettings:
     m_VersionCode: 1
     m_VersionName: 
   apiCompatibilityLevel: 6
-  activeInputHandler: 1
+  activeInputHandler: 2
```
Comment (102107466) [inline MicrowaveGame/ProjectSettings/ProjectSettings.asset:633]: Looks like it switched the project back to mixing the old and new input systems. This should be changed to new input system only.

**Review by 102107466 — CHANGES_REQUESTED** [review]
Looks good. One thing that is missing is the ability to "swap" items when you attempt to pick up an item into a slot with an existing item. Although it's not really a problem because it will just drop the item and then pick it up in two steps instead. So it can be changed later.

One thing that needs to be changed is that the input system got changed back to mixed. It should be set to use the new system only.

**Comment by 102113528** [comment]
Changes have been made and is ready for another review!

**Review by 102107466 — APPROVED** [review]
Looks good.

**Pull request merged**